stress/shell/src/main/java/alluxio/stress/cli/Benchmark.java

```diff
@@ -167,13 +167,13 @@ public String run(String[] args) throws Exception {
    *
    * @param startMs the start time
    * @param endMs the end time
-   * @param nameTransformer function which transforms the type and method into a name. If the
+   * @param javaAgentTransformer function which transforms the type and method into a name. If the
    *                        function returns null, then the method is skipped
    * @return a map of names to statistics
    */
   @SuppressFBWarnings(value = "DMI_HARDCODED_ABSOLUTE_FILENAME")
   protected Map<String, MethodStatistics> processMethodProfiles(long startMs, long endMs,
-      BiFunction<String, String, String> nameTransformer) throws IOException {
+      Function<JavaAgentInput, String> javaAgentTransformer) throws IOException {
```

Comment:
NIT: I think the term `nameTransformer` made more sense?

Reply:
changed.